docs/how-tos/backfill.rst

```diff
@@ -0,0 +1,57 @@
+Backfill old or missing data
+*****************************
+
+If you are bootstrapping a new Open edX platform with Aspects or experiencing service
+downtime in Ralph or Vector, you may need to backfill the old data. This guide will
+provide you with step-by-step instructions on how to perform the backfill process.
+
+Backfill tracking logs
```

Comment:
This should be "Backfill xAPI data from tracking logs" since we're not actually backfiling the CH stored tracking logs themselves with this process.